### ISSUE TITLE
Cleanup log entries about Alignment Memory - creating large log files

### ISSUE THREAD

**Issue by PhotoNomad0** [unfoldingWord/translationCore#6370]
### User Story

Getting lots of log entries such as:
Target Alignment Memory "n:the:servant:who:had:received:two:talents" exceeds maximum n-gram length of 5 and may be ignored.

This is creating large log files that are hard to sift through.  This is only useful for the developer trying to optimize alignment memory.  Should we remove?  should we just log a summary count?

### Features / Specifications

- [ ]
- [ ]
- [ ]

### Definition of Done

- [ ]
- [ ]
- [ ]

**Comment by deferredreward** [comment]
I've just loaded several extra projects in order to benefit from cross-project suggested alignments. Currently generating 3,000 lines of log every time I change verses.

**Comment by deferredreward** [comment]
Sorry to comment twice, but I'm not sure if I should open a second ticket for this: 

Windows Defender slows down the log write so much that it's taken over 20 minutes (before I gave up) to load another verse while I have 15 projects downloaded. In order to load another verse I had to exclude the log file from Windows Defender. (I was going to do another ticket because I didn't know why Windows Defender was causing so much slow down, but I pinpointed the log file so I'm updating here). 

Basically the verse loads but the target text areas don't seem to full load.
![image](https://user-images.githubusercontent.com/703973/64282795-27804200-cf24-11e9-9386-a834d973d459.png)

If I try to be clever and have the log open in Word (so tC can't write) I can send tC to a blank screen.
